### ISSUE TITLE
Export annotations to text format.

### ISSUE THREAD

**Issue by burtonator** [burtonator/polar-bookshelf#469]
Implement support to export annotations to text format... possibly HTML with optionally embedded data URLs

**Comment by ajvsol** [comment]
I just saw your latest blog post planning to "Include document information in exported annotations", does this mean exporting annotations is now a feature? I haven't checked out Polar in a while because that's an essential for me.

**Comment by burtonator** [comment]
Yes.. you can export the annotations for each document directly.

Also, The new annotations view will be getting a filter bar and that will also allow for more advanced exports.  You could export by tag, etc.

**Comment by ajvsol** [comment]
Ah, good to see this feature now :)

Some feedback:
- 'Comments' are not linked to the highlights they were made on 
- The actual text of each highlight could use the prefix `highlight:` or similar
- `created: 2019-02-02T21:19:22.278Z` - this could be formatted in a more readable way
- There are no line breaks between each flashcard nor highlight
- Flashcard text says "undefined" for front and back despite using one line text content for them

**Comment by burtonator** [comment]
Great.. Lots of people are asking for changes here.

https://github.com/burtonator/polar-bookshelf/blob/8447eab45ea5c47d2e359d7c33f85e9a51f30ef5/web/js/metadata/exporter/MarkdownExporter.ts

The code is super easy to play with and somewhat isolated so if anyone sends in a PR I'll probably j ust merge it.

The same code will be run from the annotations view for larger exports too.

**Comment by thiswillbeyourgithub** [comment]
How can we import annotation ? I don't see any buttons :/
